### PR TITLE
feat: prepare argos-cli for publishing

### PR DESCRIPTION
**Pull request by neoziro**

### CODE REVIEW

File: packages/argos-cli/package.json [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
   "name": "argos-cli",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "Argos command line interface.",
   "repository": "https://github.com/argos-ci/argos",
   "license": "MIT",
   "bin": {
     "argos": "bin/argos"
   },
+  "directories": [
```
Comment (oliviertassinari) [inline packages/argos-cli/package.json]: I have found that `.npmignore` allow more flexibility.

Comment (neoziro) [inline packages/argos-cli/package.json]: I want a whitelist, not a blacklist.

Comment (oliviertassinari) [inline packages/argos-cli/package.json]: /*
!/lib/*.js
!/bin/*.js
*.test.js

Comment (oliviertassinari) [inline packages/argos-cli/package.json]: I'm using https://www.npmjs.com/package/pkgfiles to be sure.

**Comment by neoziro** [comment]
https://www.npmjs.com/package/argos-cli

**Pull request merged**